### PR TITLE
More runtime cli docs

### PR DESCRIPTION
**Pull request by jafingerhut**

### CODE REVIEW

File: docs/runtime_CLI.md [hunk 1]
```diff
@@ -1,85 +1,200 @@
...
+with, and their status (e.g. up or down).
+
+
+### show_tables
+
+No parameters.  For every table in the currently loaded P4 program, shows the
+name, implementation (often `None`, but can be an action profile or action
+selector for tables created with those options), and a list of table search key
+fields, giving for each such field its name, match kind (e.g. `exact`, `lpm`,
+`ternary`, `range), and width in bits.
```
Comment (antoninbas) [inline docs/runtime_CLI.md]: missing backtick

File: docs/runtime_CLI.md [hunk 2]
```diff
@@ -1,85 +1,200 @@
...
+Match fields must appear in the same order as the table search key fields are
+specified in the P4 program, without names.  The output of `show_tables` also
+shows the names, match kinds (`ternary`, `range`, `lpm`, or `exact`), and bit
+widths for every table, after the `mk=` string.
+
+Numeric values can be specified in decimal, or hexadecimal prefixed with `0x`.
+As a special case, 32-bit values can be specified like IPv4 addresses, in dotted
+decimal, e.g. `10.1.2.3`, and 48-bit values can be specified like Ethernet MAC
+addresses, with each byte in two hexadecimal digits, with bytes separated by
+`:`, e.g. `00:12:34:56:78:9a`.
```
Comment (antoninbas) [inline docs/runtime_CLI.md]: v6 addresses can also be specified using standard IETF representation

File: docs/runtime_CLI.md (hunk 1 above)
Comment (jafingerhut) [reply docs/runtime_CLI.md]: thanks for catching that.  fixed.

File: docs/runtime_CLI.md (hunk 2 above)
Comment (jafingerhut) [reply docs/runtime_CLI.md]: I added a note about that, with some examples and a link to the relevant RFC specifying the syntax

**Pull request merged**